README.md

```diff
@@ -20,43 +12,40 @@ The Titanium CLI requires [Node.js 0.10.x](http://nodejs.org/dist/) or newer.
 
     [sudo] npm install -g titanium
 
+After install, Titanium CLI is executable as `ti`    
+
 ## Bleeding Edge
 
-You can download the latest and greatest *unstable* Titanium CLI code by
-running the following:
+You can download the latest and greatest *unstable* Titanium CLI code by running the following:
```

Comment:
This whole bleeding edge section can be removed. Nobody in the history of Titanium has ever run this command.